src/components/Tabs.jsx

```diff
@@ -7,19 +7,40 @@ import classNames from 'classnames'
 class Tabs extends React.Component {
     constructor(props, context) {
         super(props, context)
-        this.state = { activeTab: props.defaultTab || 0 }
+        const children = React.Children.toArray(this.props.children)
+
+        this.valueToTab = children.reduce((map, c, i) => {
+            const v = "value" in c.props ? c.props.value : i
```

Comment:
We're using single quotes in reactist. Running `npm run check` or even `npm run check -- --fix` should help here 👍